yaptide/simulation_runner/shieldhit_runner.py

```diff
@@ -10,95 +10,31 @@
 import pathlib
 
 from pymchelper.executor.options import SimulationSettings
-from pymchelper.executor.runner import OutputDataType, Runner
+from pymchelper.executor.runner import Runner as SHRunner
 
-input_cfg_templ = {}
-input_cfg_templ['beam.dat'] = """
-RNDSEED      	89736501     ! Random seed
-JPART0       	2            ! Incident particle type
-TMAX0      	{energy:3.6f}   0.0  ! Incident energy; (MeV/nucl)
-NSTAT       {nstat:d}    -1 ! NSTAT, Step of saving
-STRAGG          2            ! Straggling: 0-Off 1-Gauss, 2-Vavilov
-MSCAT           2            ! Mult. scatt 0-Off 1-Gauss, 2-Moliere
-NUCRE           0            ! Nucl.Reac. switcher: 1-ON, 0-OFF
-"""
-input_cfg_templ['mat.dat'] = """
-MEDIUM 1
-ICRU 276
-END
-"""
-input_cfg_templ['detect.dat'] = """
-Geometry Cyl
-    Name ScoringCylinder
-    R 0.0 10.0 {cyl_nr:d}
-    Z 0.0 30.0 {cyl_nz:d}
+from ..converter.converter.converter import DummmyParser
+from ..converter.converter.converter import Runner as ConvertRunner
```

Comment:
ConvertRunner seems wierd, wouldn't either ConvRunner or ConverterRunner be better?

Reply:
Changed